adapta/security/clients/_azure_client.py

```diff
@@ -102,6 +107,16 @@ def get_resource_group(account: StorageAccount) -> str:
                 ),
             }
 
+        if "AZURE_CLIENT_SECRET" in os.environ or "PROTEUS__AZURE_CLIENT_SECRET" in os.environ:
+            return {
+                "AZURE_CLIENT_ID": os.getenv("AZURE_CLIENT_ID") or os.getenv("PROTEUS__AZURE_CLIENT_ID"),
+                "AZURE_CLIENT_SECRET": os.getenv("AZURE_CLIENT_SECRET") or os.getenv("PROTEUS__AZURE_CLIENT_SECRET"),
+                "AZURE_TENANT_ID": os.getenv("AZURE_TENANT_ID") or os.getenv("PROTEUS__AZURE_TENANT_ID"),
```

Comment:
Maybe we should stick to only using the `PROTEUS__` env var to keep it consistent with the rest of the function

Reply:
this is a generic client and people might have azure creds mapped to env like this. We do this in airflow for flask auth to work as well, so right now we have to support both ways.